plots/wochenende_plot.py

```diff
@@ -2,526 +2,823 @@
 # coding: utf-8
```

Comment:
Are these the same changes as in #107?

Reply:
Yes, Sophia forked and pulled to dev, these are those PR and commits (with further modifications by Colin in dev) now being pulled to master. 
@poer-sophia is this correct ? Please OK this point, I think it's fine.

Reply:
I just checked, you're right. I did not see that it was a PR to dev. Sorry for the confusion